RELEASE.md

```diff
@@ -10,6 +10,7 @@
 
 ## Major features and improvements
 * Made default run entrypoint in `__main__.py` work in interactive environments such as IPyhon and Databricks.
+* Enhanced `OmegaConfigLoader` configuration validation to detect duplicate keys at all parameter levels, ensuring comprehensive nested key checking.
```

Comment:
We just released `0.19.8`, can you move this to the section above for the upcoming release? And of course same for your name below 🙂

Reply:
Thank you @merelcht ! Will do :)